app/web/themes/custom/osc/sass/uswds.scss

```diff
@@ -2,3 +2,66 @@
 @import "_variables.scss";
```

Comment:
@line47 Why remove these imports?

Reply:
@fureigh those imports were not removed they are still in the file 

link to file in `master` - https://github.com/18F/osc-website-pa/blob/master/app/web/themes/custom/osc/sass/uswds.scss